### PR TITLE
PLAT-139059: Update ThumbnailItem

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated ThumbnailItem according to the new design requirements for Silicon skin. 

### Links
[//]: # (Related issues, references)
PLAT-139059

### Comments

### CODE REVIEW

File: styles/colors-silicon-day.less [hunk 1]
```diff
@@ -355,10 +355,8 @@
 
 // ThumbnailItem
 // ---------------------------------------
-@agate-thumbnailitem-label-text-color: @agate-noble-gray;
-@agate-thumbnailitem-label-focus-text-color: @agate-noble-gray;
-@agate-thumbnailitem-label-selected-text-color: @agate-foreground;
 @agate-roundthumbnailitem-border-color: @agate-foreground;
+@agate-thumbnailitem-label-text-color:  fade(@agate-foreground, 70%);
```
Comment (adrian-cocoara-lgp) [inline styles/colors-silicon-day.less:359]: Let's change this to `#b3b3b3` to have the same color as focused and selected

File: ThumbnailItem/ThumbnailItem.module.less [hunk 2]
```diff
@@ -44,23 +45,7 @@
...
-			}
-		});
-
-		&.selected {
-			&.item {
-				.label {
-					color: @agate-thumbnailitem-label-selected-text-color;
-				}
-			}
+			font-size: @agate-thumbnailitem-content-font-size;
```
Comment (adrian-cocoara-lgp) [inline ThumbnailItem/ThumbnailItem.module.less:48]: Is this needed? Also GUI says `32px`
font-size

File: ThumbnailItem/ThumbnailItem.module.less [hunk 3]
```diff
@@ -44,23 +45,7 @@
```
Comment (adrian-cocoara-lgp) [inline ThumbnailItem/ThumbnailItem.module.less:45]: On line ~39 let's add 
`.slotBefore {
	margin-inline-end: 21px;
}` (create a silicon variable). To meet GUI requirements
margin-right

File: ThumbnailItem/ThumbnailItem.module.less [hunk 4]
```diff
@@ -7,7 +7,8 @@
   	align-items: center;
 
 	.thumbnail {
-	  	vertical-align: middle;
+		box-sizing: border-box;
+		vertical-align: middle;
```
Comment (adrian-cocoara-lgp) [inline ThumbnailItem/ThumbnailItem.module.less:11]: Is this needed?

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Review by jeonghee27 — APPROVED** [review]
LGTM. I added more screenshot TCs

**Pull request merged**